### ISSUE TITLE
Поменять названия курсов в админке на CB

### ISSUE THREAD

**Issue by Vku17** [hexlet-basics/hexlet-basics#454]
У нас есть курс по основам JS и точно такой же курс для ПК , но в админке они выглядят одинаково и не понятно, по какому в итоге у студента прогресс. Можно ли по этому курсу и другим одноименным сделать различающиеся названия
![image](https://github.com/user-attachments/assets/e87c7224-6c62-4513-9002-d340b0c62bf7)

**Comment by mokevnin** [comment]
@fey да, поменяйте плс хедеры в репе у доп курсов и все будет хорошо
